documentation/tutorials/using_the_voting_cli.md

```diff
@@ -0,0 +1,79 @@
+# Voting Command Line Interface (CLI)
+
+We provide a user-friendly command line interface for voting on price requests and retrieving rewards.
+
+# Setup
```

Comment:
One super minor nit: a heading followed immediately by a subheading doesn't look great (and is a bit confusing) when rendered on the docs site: https://28293-142203027-gh.circle-artifacts.com/0/build/site/uma/tutorials/using_the_voting_cli.html#_setup. I would recommend either adding a little text between to introduce the `Setup` section or promoting the subheadings and removing `Setup` altogether.

Reply:
done